google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/wire/PartitionCountWatchingPublisherBuilder.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Don't add new builders in this style if you could? Could you instead make this a PartitionCountWatchingPublisherSettings class with an instantiate() method?

Reply:
Ah, yeah, that pattern makes more sense to me. I was just copying the structure of other builders in the directory